models/users.js

```diff
@@ -59,6 +59,7 @@ const addOrUpdate = async (userData, userId = null) => {
 const fetchUsers = async (query) => {
   try {
     const snapshot = await userModel
+      .where("isMember", "==", true)
```

Comment:
We do not have to return users with Members role by default, instead we should filter the users based on role provided as query params by the client and in case no role is provided we return the data of all the users.
Eg: `/users?role=members`